### ISSUE TITLE
[APP SUBMITTED]: database or disk is full

### ISSUE THREAD

**Issue by duramato** [pymedusa/SickRage#704]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@881d1224faaaaecd4228d9572f749044f222fe2a
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-250793 :: [BeyondHD] :: [881d122] DB error: database or disk is full
Traceback (most recent call last):
  File "/home/home/SickRage/sickbeard/db.py", line 254, in action
    sql_results = self._execute(query, args, fetchall=fetchall, fetchone=fetchone)
  File "/home/home/SickRage/sickbeard/db.py", line 108, in _execute
    sql_results = self.connection.cursor().execute(query, args)
OperationalError: database or disk is full
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by duramato** [comment]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@881d1224faaaaecd4228d9572f749044f222fe2a
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-250793 :: [SickBeard Index] :: [881d122] DB error: database or disk is full
Traceback (most recent call last):
  File "/home/home/SickRage/sickbeard/db.py", line 254, in action
    sql_results = self._execute(query, args, fetchall=fetchall, fetchone=fetchone)
  File "/home/home/SickRage/sickbeard/db.py", line 108, in _execute
    sql_results = self.connection.cursor().execute(query, args)
OperationalError: database or disk is full
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by duramato** [comment]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@881d1224faaaaecd4228d9572f749044f222fe2a
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-250793 :: [LimeTorrents] :: [881d122] DB error: database or disk is full
Traceback (most recent call last):
  File "/home/home/SickRage/sickbeard/db.py", line 254, in action
    sql_results = self._execute(query, args, fetchall=fetchall, fetchone=fetchone)
  File "/home/home/SickRage/sickbeard/db.py", line 108, in _execute
    sql_results = self.connection.cursor().execute(query, args)
OperationalError: database or disk is full
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by duramato** [comment]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@881d1224faaaaecd4228d9572f749044f222fe2a
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-250793 :: [PrivateHD] :: [881d122] DB error: database or disk is full
Traceback (most recent call last):
  File "/home/home/SickRage/sickbeard/db.py", line 254, in action
    sql_results = self._execute(query, args, fetchall=fetchall, fetchone=fetchone)
  File "/home/home/SickRage/sickbeard/db.py", line 108, in _execute
    sql_results = self.connection.cursor().execute(query, args)
OperationalError: database or disk is full
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by duramato** [comment]
### INFO
Python Version: **2.7.6 (default, Jun 22 2015, 18:00:18) [GCC 4.8.2]**
Operating System: **Linux-3.19.0-56-generic-i686-with-Ubuntu-14.04-trusty**
Locale: UTF-8
Branch: **develop**
Commit: PyMedusa/SickRage@881d1224faaaaecd4228d9572f749044f222fe2a
No Log available with ERRORS:
### ERROR
```
SEARCHQUEUE-BACKLOG-250793 :: [Nutech] :: [881d122] DB error: database or disk is full
Traceback (most recent call last):
  File "/home/home/SickRage/sickbeard/db.py", line 254, in action
    sql_results = self._execute(query, args, fetchall=fetchall, fetchone=fetchone)
  File "/home/home/SickRage/sickbeard/db.py", line 108, in _execute
    sql_results = self.connection.cursor().execute(query, args)
OperationalError: database or disk is full
```
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by duramato** [comment]
When the disk is full , this happens which totally kills the threads.

**Comment by OmgImAlexis** [comment]
I also get this when using UNRAID as I have a SSD cache setup so once the cache gets filled it assumes there's no space and doesn't move anything else even though as soon as it does the cache will empty old files out. 😞

**Comment by Arcanemagus** [comment]
UNRAID's cache isn't transparent? lol wtf...

**Comment by ratoaq2** [comment]
Fixed by #817 (develop branch)
